bin/install_etf.sh

```diff
@@ -0,0 +1,259 @@
+#!/bin/sh
+#############################################################################
+#
+# Purpose: This script will install INSPIRE ETF
+# Author:
+# Version 2020-08-28
+#
+#############################################################################
+# Copyright (c) 2011-2019 The Open Source Geospatial Foundation and others.
+# Licensed under the GNU LGPL.
+#
+# This library is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published
+# by the Free Software Foundation, either version 2.1 of the License,
+# or any later version.  This library is distributed in the hope that
+# it will be useful, but WITHOUT ANY WARRANTY, without even the implied
+# warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Lesser General Public License for more details, either
+# in the "LICENSE.LGPL.txt" file distributed with this software or at
+# web page "http://www.fsf.org/licenses/lgpl.html".
+#
+#############################################################################
+from cloudinit.util import chmod
+
+# =============================================================================
+# Install script for ETF
+# =============================================================================
+#
+# Variables
+# -----------------------------------------------------------------------------
+START=$(date +%M:%S)
+./diskspace_probe.sh "`basename $0`" begin
+BUILD_DIR=`pwd`
+TMP="/tmp/build_etf"
+if [ -z "$USER_NAME" ] ; then
+   USER_NAME="user"
+fi
+USER_HOME="/home/$USER_NAME"
+JETTY9_SCRIPT_NAME="jetty9"
+ETF_WEB_APP_NAME="ETF"
+ETF_ICON_NAME="ETF_logo.png" #to be changed to the INSPIRE logo
+ETF_URL="http://localhost:9090/ETF"
+ETF_PORT="9090"
+ETF_FOLDER="$USER_HOME/.etf"
+ETF_WAR_INSTALL_FOLDER="/usr/share/$JETTY9_SCRIPT_NAME/webapps"
+ETF_BIN_FOLDER="/usr/local/share/ETF"
+ETF_VERSION="2.0"
+JAVA_PKG="openjdk-8-jdk"
```

Comment:
please use openjdk-8-jre-headless

Reply:
changed to use openjdk-8-jdk-headless and small modifications have been made